README.md

```diff
@@ -46,6 +46,17 @@ make install ## Required to have go installed in the machine
 stevedore --help ## Verify the installation
 ```
 
+### Use Docker image
+```
+alias stevedore="docker run -it -v ~/.config/stevedore:/root/.config/stevedore -v ~/.kube:/root/.kube -v $PWD:/workdir thecasualcoder/stevedore"
```

Comment:
Once we have released the binary, we will be able to provide use [godownloader](https://github.com/goreleaser/godownloader) to install stevedore easily